CHANGELOG.md

```diff
@@ -9,6 +9,35 @@ Unreleased section should follow [Release Toolkit](https://github.com/newrelic/r
 
 ## Unreleased
 
+### enhancement
+- Add metrics collection from DockerAPI to support nodes with cgroup V2 on ECS clusters.
+
+### bugfix
+- Blkio metrics `TotalReadCount`, `TotalWriteCount`, `TotalReadBytes` and `TotalWriteBytes` will not be sent if not present. Before they were being reported with value `0` (#221)
```

Comment:
Should we move this to `breaking`? As I see it, it would be the same that the swap metrics bug-fix.

Reply:
I think there's a difference since swap metrics were being badly reported on Fargate, not with 0 values. Another thing is if we consider this a breaking change or not... we comment on it briefly [here](https://github.com/newrelic/nri-docker/pull/228#discussion_r1519803121). As said, I don't have a strong opinion in this case.